common/modifiers/wc_event_modifiers.txt

```diff
@@ -313,4 +313,22 @@ wc_thunder_king_ai_modifier = { # AI Thunder King only
 #wc_troll_mogu_pact_thunder_king_modifier = { # Thunder King/ Zandalari Pact handling
 #	icon = martial_positive
 #	monthly_prestige = 0.2
-#}
\ No newline at end of file
+#}
+
+# Thorium Brotherhood modifiers
+wc_thorium_craftsmen_modifier = {
+	icon = county_modifier_development_positive
+	development_growth_factor = 0.1
+	tax_mult = 0.05
+	build_gold_cost = -0.15
+	build_speed = -0.15
+}
+
+wc_thorium_base_modifier = {
+	icon = martial_positive
```

Comment:
Martial icon?

Reply:
it adds mostly martial bonuses, can change if necessary

Reply:
changed